ipa-core/src/protocol/prss/mod.rs

```diff
@@ -41,17 +44,15 @@ impl UsedSet {
     /// ## Panics
     /// Panic if this index has been used before.
     fn insert(&self, index: PrssIndex128) {
-        let index = u128::from(index);
-        if index > usize::MAX as u128 {
-            tracing::warn!(
-                "PRSS verification can validate values not exceeding {}, index {index} is greater.",
-                usize::MAX
-            );
+        let raw_index = u128::from(index);
+        if raw_index > usize::MAX as u128 {
+            // This is unreachable with the current PRSS index encoding.
+            tracing::warn!("Index '{index}' is not supported by PRSS verification.");
```

Comment:
I may prefer the previous message that specifies the bounds because if I ever see it in the logs, I wouldn't need to go to the code to see why it was rejected

Reply:
```suggestion
            tracing::warn!("PRSS index is too large: {index} > usize::MAX");
```